### PR TITLE
Merge v0.39.0_rc to master

### PR DESCRIPTION
**Pull request by multiphaseCFD**
### Before submitting

Please complete the following checklist when submitting a PR:

- [ ] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      [`tests`](../tests) directory!

- [ ] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [ ] Ensure that the test suite passes, by running `make test`.

- [x] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
      change, and including a link back to the PR.

- [x] Ensure that code is properly formatted by running `make format`. 

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:**

**Description of the Change:**

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -1,100 +1,130 @@
...
+### Documentation
+
+### Bug fixes
+
+### Contributors
 
-* Add `Projector` observable support via diagonalization to Lightning-GPU.
+This release contains contributions from (in alphabetical order):
+
+Shuli Shu
```
Comment (maliasadi) [inline .github/CHANGELOG.md:20]:
```suggestion
```

**Review by maliasadi — APPROVED** [review]
:partying_face:

File: .github/CHANGELOG.md [hunk 2]
```diff
@@ -1,100 +1,129 @@
-# Release 0.39.0-dev
+# Release 0.40.0-dev0
```
Comment (maliasadi) [inline .github/CHANGELOG.md:1]:
```suggestion
# Release 0.40.0-dev
```

**Review by LuisAlfredoNu — APPROVED** [review]
LGTM
Thank you @multiphaseCFD

**Pull request merged**